aspnetcore/tutorials/razor-pages/razor-pages-start/snapshot_sample5/RazorPagesMovie30/Pages/Movies/Delete.cshtml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using RazorPagesMovie.Data;
+using RazorPagesMovie.Models;
+
+namespace RazorPagesMovie.Pages.Movies
+{
+    public class DeleteModel : PageModel
+    {
+        private readonly RazorPagesMovie.Data.RazorPagesMovieContext _context;
+
+        public DeleteModel(RazorPagesMovie.Data.RazorPagesMovieContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Movie Movie { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            Movie = await _context.Movie.FirstOrDefaultAsync(m => m.ID == id);
```

Comment:
```suggestion
            Movie = await _context.Movie.FindAsync(id);
```

Reply:
It compiles and runs, however, the code already there is what the scaffolding generates, and we want the user in this case to use what the scaffolding creates.

So I will not make the change.  However, I think you have a good argument for a small improvement in the scaffolding code of the product.  :)